zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -364,6 +370,7 @@ impl<F: Field> ExecutionConfig<F> {
             caller_gadget: configure_gadget!(),
             coinbase_gadget: configure_gadget!(),
             comparator_gadget: configure_gadget!(),
+            copy_to_log_gadget: configure_gadget!(),
```

Comment:
move this to internal state section.

Reply:
update in [755a06d](https://github.com/appliedzkp/zkevm-circuits/pull/335/commits/755a06d429eb994510af696e5c89d0c96f086176) thanks